app/views/order_items/cart.html.erb

```diff
@@ -0,0 +1,94 @@
+<% if @instock.length == 1 %>
```

Comment:
No logic in views!

This is a great place to have a helper method that returns the boolean value you want.